app/controllers/users/projects_controller.rb

```diff
@@ -3,4 +3,22 @@ class Users::ProjectsController < ApplicationController
   def new
     @user = current_user
   end
+
+  def create
+    portfolio = User.find(params[:user_id]).portfolio
+    new_project = portfolio.projects.new(project_params)
+    new_project.update(user_id: params[:user_id])
+    if(new_project.save)
+      sleep(1)
+      render js: "/dashboard"
+    else
+      render component: 'ProjectForm', props: { user: current_user}
+    end
+  end
```

Comment:
This seems like a bit of a mess.

1. Why is there a sleep?
2. Why are you rendering instead of redirecting?

Is this just the bug from "Redirect after creating a profile"?